pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -42,8 +42,8 @@ type Service struct {
 }
 
 // New return didexchange service
-func New(store storage.Store, prov provider) *Service {
-	return &Service{outboundTransport: prov.OutboundTransport(), store: store}
+func New(store storage.Store, OutboundProv provider) *Service {
```

Comment:
any specific reason to change this argument name ?

Reply:
done